centralstore/redis_store.go

```diff
@@ -264,48 +264,57 @@ func (r *RedisBasicStore) WriteSpans(ctx context.Context, spans []*CentralSpan)
 	return nil
 }
 
-// GetTrace returns a CentralTrace with the given traceID.
+// GetTraces returns a CentralTrace with the given traceID.
 // if a decision has been made about the trace, the returned value
 // will not contain span data.
-func (r *RedisBasicStore) GetTrace(ctx context.Context, traceID string) (*CentralTrace, error) {
+func (r *RedisBasicStore) GetTraces(ctx context.Context, traceIDs ...string) ([]*CentralTrace, error) {
 	_, span := r.Tracer.Start(ctx, "GetTrace")
 	defer span.End()
 
 	conn := r.RedisClient.Get()
 	defer conn.Close()
 
-	spans := make([]*CentralSpan, 0)
-	var tmpSpan []struct {
-		SpanID string
-		Span   []byte
+	for _, traceID := range traceIDs {
+		cmd := redis.NewGetAllValuesHashCommand(spansHashByTraceIDKey(traceID))
+		if err := cmd.Send(conn); err != nil {
+			return nil, err
+		}
 	}
 
-	err := conn.GetSliceOfStructsHash(spansHashByTraceIDKey(traceID), &tmpSpan)
+	data, err := conn.ReceiveByteSlice(len(traceIDs))
```

Comment:
Given that this returns `[][][]byte`, I feel like I would like to see a lot more comments in the loops here explaining the logic here. 

Also, shouldn't this be called `ReceiveByteSlices`?

(Really, I wish there were a better way to describe that type, but you're stuck with what redigo did.)